zproject/urls.py

```diff
@@ -549,7 +549,7 @@
     ),
     path("accounts/deactivated/", show_deactivation_notice),
     # Displays digest email content in browser.
-    path("digest/", digest_page),
+    path("digest/", digest_page, name="digest_page"),
```

Comment:
Do we need the `name` parameter here?

Reply:
I guess we do since we access this page via `url()` above.

Reply:
Yeah.